homework-02/task-01.js

```diff
@@ -0,0 +1,7 @@
+const logItems = function (array) {
+  for (let i = 0; i < array.length; i++) {
+    console.log(i + 1, array[i]);
```

Comment:
тут потрібно було використати "template strings" :) 
```suggestion
    console.log(`${i + 1}:  ${array[i]}`);
```